rudder-agent/SOURCES/rudder-agent.init

```diff
@@ -225,7 +236,7 @@ start_daemons() {
 
 		# Is daemon launched?
 		PID=`cat ${CFENGINE_COMMUNITY_PID_FILE[$daemon]}`
-		if [ ! -e /proc/$PID ]
+		if ! ps -p "${PID}" > /dev/null
```

Comment:
Why are you changing this here? Is /proc/$PID not available on all systems the reason? Or is it related to this bug?